x-pack/plugins/security_solution/public/detection_engine/rule_details_ui/pages/rule_details/index.tsx

```diff
@@ -791,6 +799,12 @@ const RuleDetailsPageComponent: React.FC<DetectionEngineComponentProps> = ({
                         )}
                       </StepPanel>
                     </EuiFlexItem>
+                    {ruleActionsData != null && (
+                      <StepActionsRule
```

Comment:
I think we should be using `StepRuleActionsReadOnlyComponent` component here to follow the same pattern that we use for the rest of the rule steps. Also, I think we could add all your changes from the `StepActions` inside the `StepRuleActionsReadOnlyComponent`. I don't see the benefit of having a separate file for that functionality, the `StepRuleActionsReadOnlyComponent` was made for it :-)